packages/graphql-toolbox/README.md

```diff
@@ -45,6 +45,20 @@ Run this command to start the development server. See the output in the console
 yarn start
 ```
 
+## GraphQL Toolbox URL parameters
+
+The GraphQL Toolbox supports some URL parameters defined in the query component, see below. These URL parameters are optional and can be provided at convenience.
+The query component is preceded by a question mark (?) and contains a query string that is a sequence of key-value pairs separated by an ampersand (&).
```

Comment:
```suggestion
The GraphQL Toolbox supports some URL parameters defined in the query string, see below. These URL parameters are optional and can be provided for convenience.
The query string is preceded by a question mark (?) and contains key-value pairs separated by an ampersand (&).
```

I think with this change the language becomes more standardised, so more easily understandable.